config/manager/manager.yaml

```diff
@@ -21,6 +21,7 @@ spec:
     metadata:
       labels:
         control-plane: controller-manager
+        app: limitador
```

Comment:
Just to clarify, this is the label on the operator pod deployment. Not the operand (limitador instance). 

The labels on the  operand are defined in the go code (as the limitador deployment is coded in go) https://github.com/Kuadrant/limitador-operator/blob/4669d072dcdbb8cb5c1285196e5c52ab4570af9c/pkg/limitador/k8s_objects.go#L219-L224 which happens to be the same `app: limitador`. 

Do we want to have the same label in operator and operand? If the answer is, "yes, it does not matter". you have  my LGTM.

Reply:
Yes its a label for the operator pod deployment not the operand. In the scnerio that i want it for it does not matter I'm also using another label along with it in the selector but to avoid confusion i can change it to be different

Reply:
thinking further it would mean actually i would just need the one label. Ok yeah let me do that

Reply:
then LGTM

Reply:
Ok ive updated to app: limitador-operator, happy with that instead?

Reply:
I was happy with `app: limitador`. It is about you guys :)